wa/framework/getters.py

```diff
@@ -80,7 +80,12 @@ def get_from_location(basepath, resource):
         path = os.path.join(basepath, 'bin', resource.abi, resource.filename)
         if os.path.exists(path):
             return path
-    elif resource.kind in ['apk', 'jar', 'revent']:
+    elif resource.kind == 'revent':
+        path = os.path.join(basepath, 'revent_files')
+        if os.path.exists(path):
```

Comment:
This needs to check in both locations. if the file is not found in "revent_files" subdirectory, look in the base resource directory.